README.md

```diff
@@ -202,6 +213,14 @@ properties.
 
   The size of the generated archive, in bytes.
 
+- `isInternalFunction` `boolean`
```

Comment:
Could this be just `internal`?

Reply:
I'd then maybe change it to `internalFunction` just because I like it to point explicitly to _what_ it is that is internal :) I think `internal` might just be a bit too implicit, and anything to help understand what it refers to might be nice.

Reply:
But all of the properties in the output object are about a specific function, so rather than `path` and `runtime` you'd have `functionPath`, `functionRuntime`, etc.

Reply:
That's true 🤔 yeah I'll change it!

Reply:
How about `isInternal`. I know it looks like a small difference, but then it at least still refers to itself somehow? Giving it a bit more context?

Reply:
Go for it.